lib/Doctrine/ODM/PHPCR/UnitOfWork.php

```diff
@@ -1437,8 +1438,13 @@ private function computeChildrenChanges($document, $class, $oid, $isNew, $change
                 // check moved children to not accidentally remove a child that simply moved away.
                 if (!(in_array($childName, $childNames) || in_array($childName, $movedChildNames))) {
```

Comment:
does the second condition still make sense, or is it covered by checking in scheduled moves?

Reply:
Nope, its not covered, the move is scheduled afterwards, so both have to exist for now, I think thats why it was detected like this initially